hyperon_das/das.py

```diff
@@ -20,6 +20,29 @@
 
 class DistributedAtomSpace:
     def __init__(self, system_parameters: Dict[str, Any] = {}, **kwargs) -> None:
+        """Creates a new DAS object
+
+        Args:
+            system_parameters (Dict[str, Any], optional): Sets the system parameters. 
+                Defaults to {'running_on_server': False, 'cache_enabled': False, 'attention_broker_hostname': 'localhost', 'attention_broker_port': 27000}.
+
+        Keyword Args:
+            atomdb (str, optional): AtomDB type suported values are 'ram' and 'redis_mongo'. Defaults to 'ram'
```

Comment:
The period is missing at the end of the paragraph. Please check all the following sentences because there are other ones with the same typo.